setup.py

```diff
@@ -57,8 +61,13 @@ def read(filepath):
         'zope.security',
     ],
     extras_require=dict(
+        mruwidget=[
+            'zope.annotation',
+            'BTrees',
+            'persistent',
+            'zc.resourcelibrary',
+        ],
         test=[
```

Comment:
I wonder if it would be possible to make the `[test]` extra depend on `zc.form[mruwidget]`, so tox.ini and .travis.yml could  continue using just the `[test]`.